experiments/acic18/script.R

```diff
@@ -46,6 +47,9 @@ cf.raw = causal_forest(X, Y, W,
 varimp = variable_importance(cf.raw)
 selected.idx = which(varimp > mean(varimp))
 
+# Currently failing with error:
```

Comment:
if we have the package version check up top, I believe we don't need this comment

Reply:
Yes, that was just my personal note!